src/accounts.js

```diff
@@ -256,14 +256,22 @@ async function checkIfAccountHasPermission(oreAccountName, permName) {
   return !!(perms.find(perm => perm.perm_name === permName));
 }
 
-async function addPermission(authAccountName, keys, permissionName, parentPermission, options = {}) {
+async function addPermission(authAccountName, keys, permissionName, parentPermission, overridePermission = false, options = {}) {
```

Comment:
Not usually a good practice to add a parameter that breaks the existing interface. At a minimum, if we do this, we should change the version number to be a breaking release (e.g. 1.9.0)

Reply:
i had to change the function interface only for this function. This was initially appending a permission to an existing. But for a new account feature, we need it to override the existing permission. Changing the version number to 1.9.0